mocha.js

```diff
@@ -776,7 +776,7 @@ module.exports = function(suite) {
   });
```

Comment:
If you take a look at the commit log you will see that Mocha is being build before release by the maintainer who releases it.